pkg/core/interop_neo_crypto.go

```diff
@@ -29,3 +32,159 @@ func (ic *interopContext) bytesToPublicKey(b []byte) (*keys.PublicKey, error) {
 	}
 	return pub, nil
 }
+
+// ecdsaCheckMultisig checks multiple ECDSA signatures at once.
+func (ic *interopContext) ecdsaCheckMultisig(v *vm.VM) error {
+	msg, err := v.Estack().Pop().Item().TryBytes()
+	if err != nil {
+		return err
+	}
+	hashToCheck := hash.Sha256(msg).BytesBE()
+	pkeys, err := v.Estack().PopSigElements()
+	if err != nil {
+		return fmt.Errorf("wrong parameters: %s", err.Error())
+	}
+	sigs, err := v.Estack().PopSigElements()
+	if err != nil {
+		return fmt.Errorf("wrong parameters: %s", err.Error())
+	}
+	// It's ok to have more keys than there are signatures (it would
+	// just mean that some keys didn't sign), but not the other way around.
+	if len(pkeys) < len(sigs) {
+		return errors.New("more signatures than there are keys")
+	}
+	sigok, err := checkMultisigPar(ic, hashToCheck, pkeys, sigs)
+	if err != nil {
+		return err
+	}
+	v.Estack().PushVal(sigok)
+	return nil
+}
+
+func checkMultisigPar(ic *interopContext, h []byte, pkeys [][]byte, sigs [][]byte) (bool, error) {
```

Comment:
It's better to make the function in `vm` reusable for now, we can move it later if need be.

Reply:
FIxed.